modin/pandas/__init__.py

```diff
@@ -195,6 +195,8 @@ def initialize_ray():
 
             client = Client()
         num_cpus = sum(client.ncores().values())
+elif execution_engine == "Multiprocess":
+    num_cpus = 4    # todo: need to calc this
```

Comment:
```suggestion
    import multiprocessing

    num_cpus = multiprocessing.cpu_count()
```
That's how ray gets the cpu count if none is provided